packages/react-charting/src/components/GaugeChart/__snapshots__/GaugeChart.test.tsx.snap

```diff
@@ -4900,7 +4878,6 @@ exports[`GaugeChart snapshot tests should render the chart value in fraction for
           data-is-focusable={true}
           fill="#107c10"
           fillOpacity={1}
-          key="0"
```

Comment:
why is key showing as removed

Reply:
these `<path>` elements were wrapped in a `<React.Fragment>` in code. The key was moved to the fragment to satisfy the key requirement for `arcs.map()`. 

Same for comment on GroupedVerticalBarChart snapshot

Reply:
Sounds fine.